File: coco/src/main/java/coco/ep/m7i2/Add.java
package coco.ep.m7i2;

import coco.ep.Exp;

public interface Add<FT> extends coco.ep.m7.Add<FT>, coco.ep.i1.Add<FT>, coco.ep.m7i2.Exp<FT>, Factory<FT> {

    default Exp<FT> powby(Exp<FT> other) {
        return convert(this.lit(convert(this).eval())).powby(convert(other));
    }
}

File: coco_csharp/CoCo/EP/M7/Add.cs
namespace EP.M7 {
    public interface Add<FT> : Exp<FT>, EP.M6.Add<FT> {
        EP.Exp<FT> Exp<FT>.powby(EP.Exp<FT> other) {
            return convert(lit(convert(this).eval())).powby(convert(other));
        }
    }
}